google/cloud/examples/BUILD

```diff
@@ -41,15 +47,49 @@ cc_test(
     deps = [":gcs2cbt_lib"],
 )
 
+proto_library(
+    name = "hello_world_protos",
+    srcs = [
+        "hello_world_grpc/hello_world.proto",
+    ],
+    visibility = [
+        "//:__pkg__",
+    ],
+    deps = [],
+)
+
+cc_proto_library(
+    name = "hello_world_cc_proto",
+    visibility = [
+        "//:__pkg__",
+    ],
+    deps = [":hello_world_protos"],
+)
+
+cc_grpc_library(
+    name = "hello_world_cc_grpc",
+    srcs = [":hello_world_protos"],
+    grpc_only = True,
+    visibility = [
+        "//:__pkg__",
```

Comment:
I think this visibility rule is granting access to this target from the targets defined in our top-level `BUILD` file, is that the intent? If the goal is to simply make this target private to the current package (i.e. `//google/cloud/example`), then I think you may want `//visibility:private`

Reply:
Fixed.